src/test/e2e/api/admin/applications.e2e.test.ts

```diff
@@ -0,0 +1,132 @@
+import dbInit, { ITestDb } from '../../helpers/database-init';
+import {
+    IUnleashTest,
+    setupAppWithCustomConfig,
+} from '../../helpers/test-helper';
+import getLogger from '../../../fixtures/no-logger';
+import {
+    ApiTokenType,
+    IApiToken,
+} from '../../../../lib/types/models/api-token';
+
+let app: IUnleashTest;
+let db: ITestDb;
+let defaultToken: IApiToken;
+
+const metrics = {
+    appName: 'appName',
+    instanceId: 'instanceId',
+    bucket: {
+        start: '2016-11-03T07:16:43.572Z',
+        stop: '2016-11-03T07:16:53.572Z',
+        toggles: {
+            'toggle-name-1': {
+                yes: 123,
+                no: 321,
+                variants: {
+                    'variant-1': 123,
+                    'variant-2': 321,
+                },
+            },
+            'toggle-name-2': {
+                yes: 123,
+                no: 321,
+                variants: {
+                    'variant-1': 123,
+                    'variant-2': 321,
+                },
+            },
+            'toggle-name-3': {
+                yes: 123,
+                no: 321,
+                variants: {
+                    'variant-1': 123,
+                    'variant-2': 321,
+                },
+            },
+        },
+    },
+};
+
+beforeAll(async () => {
+    db = await dbInit('applications_serial', getLogger, {});
+    app = await setupAppWithCustomConfig(
+        db.stores,
+        {
+            experimental: {
+                flags: {
+                    strictSchemaValidation: true,
+                    sdkReporting: true,
+                },
+            },
+        },
+        db.rawDatabase,
+    );
+
+    defaultToken =
+        await app.services.apiTokenService.createApiTokenWithProjects({
+            type: ApiTokenType.CLIENT,
+            projects: ['default'],
+            environment: 'default',
+            tokenName: 'tester',
+        });
+});
+
+afterEach(async () => {
+    await db.stores.clientMetricsStoreV2.deleteAll();
```

Comment:
I've never noticed this before but maybe Promise.all is possible here

Reply:
I used promise all and the benefit is not existant. But in theory it will help, so I added this change.